airflow/providers/google/marketing_platform/example_dags/example_display_video.py

```diff
@@ -53,6 +55,18 @@
 PARAMS = {"dataRange": "LAST_14_DAYS", "timezoneCode": "America/New_York"}
 # [END howto_display_video_env_variables]
 
+# download_line_items variables
+ADVERTISER_ID = os.environ.get("GMP_ADVERTISER_ID", 1234567)
+BUCKET_NAME = "AWESOME_BUCKET"
+OBJECT_NAME = "HOW_TO_CALL_THIS_OBJJECT"
+GZIP = False
+FETCH_ITEM = {
```

Comment:
Is that how it's named in API docs?

Reply:
No, just body. I will rename it.

http://googleapis.github.io/google-api-python-client/docs/dyn/doubleclickbidmanager_v1.lineitems.html#downloadlineitems

Reply:
Probably it should be `LINE_ITEM`